runtime/bcverify/staticverify.c

```diff
@@ -122,7 +122,12 @@ buildInstructionMap (J9CfrClassFile * classfile, J9CfrAttributeCode * code, U_8
 			break;
 
 		default:
-			if (bc > CFR_BC_jsr_w) {
+#if defined(J9VM_OPT_VALHALLA_VALUE_TYPES)
+			if ((bc > CFR_BC_withfield) || (bc == CFR_BC_breakpoint))
```

Comment:
This will be broken if new opcode greater than `CFR_BC_withfield` is added in the future. Maybe you can define sth like `CFR_BC_MaxDefined`, set it to `CFR_BC_withfield` (for VT) or `CFR_BC_jsr_w` (for non-VT). Compare against `CFR_BC_MaxDefined` here.